Helper/Api/Orders.php

```diff
@@ -104,6 +107,7 @@ public function __construct(
      */
     public function prepareOrder(Order $order, bool $isUpdate): array
     {
+        $this->order = $order;
```

Comment:
```suggestion
```

On all other places in this helper, we pass the `Order` object, I think it currency should be passed as well not saved.